vulkaninfo/vulkaninfo.rc.in

```diff
@@ -45,8 +46,8 @@ BEGIN
         BEGIN
             VALUE "FileDescription", VER_FILE_DESCRIPTION_STR
             VALUE "FileVersion", VER_FILE_VERSION_STR
-            VALUE "LegalCopyright", "Copyright (C) 2015-2021"
-            VALUE "ProductName", "Vulkan Runtime"
+            VALUE "LegalCopyright", VER_COPYRIGHT_STR
+            VALUE "ProductName", "Vulkaninfo"
```

Comment:
should this be capitalized?

Reply:
What should be capitalized exactly?

Reply:
```
            VALUE "ProductName", "vulkaninfo"
```

Reply:
Seems a bit weird to call it VULKANINFO, but I suppose you mean vulkaninfo (no leading capital letter), which looks better to me anyhow.